gitnexus/test/e2e/test_analyze_kotlin_grammar.py

```
@@ -0,0 +1,36 @@
+FIXTURE = PACKAGE / "test" / "fixtures" / "lang-resolution" / "kotlin-overload-dispatch"
+
+
+class AnalyzeKotlinGrammarTests(unittest.TestCase):
+    def setUp(self) -> None:
+        self.home = Path(tempfile.mkdtemp(prefix="gitnexus-kotlin-"))
+        self.addCleanup(shutil.rmtree, self.home, True)
+        self.repo = self.home / "repo"
+        shutil.copytree(FIXTURE, self.repo)
+
+    def cli(self, *args: str) -> subprocess.CompletedProcess[str]:
+        return subprocess.run([*SOURCE_ENTRY, *args], cwd=PACKAGE, text=True, capture_output=True,
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)